typescript/src/enums.ts

```diff
@@ -244,3 +244,45 @@ export enum PURCHASES_ARE_COMPLETED_BY {
    */
   REVENUECAT = "REVENUECAT",
 }
+
+/**
+ * Configuration option that specifies that your app will complete purchases.
+ * @public
+ */
+export type PurchasesAreCompletedByMyApp = {
+  type: PURCHASES_ARE_COMPLETED_BY_TYPE.MY_APP;
+
+  /**
+   * The version of StoreKit that your app is using to make purchases. This value is ignored
+   * on Android, so if your app is Android-only, you may provide any value.
+   */
+  storeKitVersion: STOREKIT_VERSION;
```

Comment:
I guess with this approach there is a chance of people setting conflicting versions here and in the `storeKitVersion` parameter in the `PurchasesConfiguration`... IMO in those cases we should just log a warning and default to this one.

Reply:
Great point - I'll be sure to do that in the hybrid SDKs 😄